_integration-schemas/ms-teams/v1/team_device_usage_report.md

```diff
@@ -0,0 +1,65 @@
+---
+tap: "ms-teams"
+version: "1"
+key: ""
+
+name: "team_device_usage_report"
+doc-link: "https://docs.microsoft.com/en-us/graph/api/reportroot-getteamsdeviceusageuserdetail?view=graph-rest-beta"
+singer-schema: "https://github.com/singer-io/tap-ms-teams/blob/master/tap_ms_teams/schemas/team_device_usage_report.json"
+description: |
+  The `{{ table.name }}` table contains information about a group's device usage in {{ integration.display_name }}.
+
+replication-method: "Key-based Incremental"
+
+api-method:
+    name: |
+      reportRoot: getTeamsDeviceUsageUserDetail
+    doc-link: "https://docs.microsoft.com/en-us/graph/api/reportroot-getteamsdeviceusageuserdetail?view=graph-rest-beta"
+    
+attributes:
+  - name: "id"
+    type: "string"
+    primary-key: true
+    description: "The device ID."
+    #foreign-key-id: "device-id"
```

Comment:
See my answer to your questions - this table actually has two Primary Keys, neither of which is `id` :)